tooling/template-base/components/LearnerAccess/DisplayListView.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import dashboardDefault from '../../renderer/dashboardDefault.png';
+import dropDownClosed from '../../renderer/dropDownClosed.svg';
+import dropDownOpen from '../../renderer/dropDownOpen.svg';
+import LearnerAccessListDisplayDropDown from './ListDisplayDropDown';
+import {
+  HydratedContentItem,
+  useUserCourseCompletionProgressQuery
+} from '@thoughtindustries/content';
+
+interface ContentUiProps {
+  item: HydratedContentItem;
+  index?: number;
+}
+
+const LearnerAccessDisplayListView = ({ item }: ContentUiProps) => {
+  // const listViewDropDown = useContext(ListViewDropDown)
+  const [listViewDropDown, setListViewDropDown] = useState(false);
+
+  const { data } = useUserCourseCompletionProgressQuery({
```

Comment:
We'll also want to pass `fetchPolicy: 'network-only'` to the object passed to `useUserCourseCompletionProgressQuery` to ensure we're not getting cached results. Same for other graphql queries.